src/ts/components/subtitlesettings/subtitlesettingspanelpage.ts

```diff
@@ -16,16 +14,19 @@ import {WindowOpacitySelectBox} from './windowopacityselectbox';
 import {SubtitleSettingsResetButton} from './subtitlesettingsresetbutton';
 import {UIInstanceManager} from '../../uimanager';
 import {SettingsPanelPageBackButton} from '../settingspanelpagebackbutton';
-import {SettingsPanelItem} from '../settingspanelitem';
+import { SettingsPanelItem, SettingsPanelItemConfig } from '../settingspanelitem';
 import { PlayerAPI } from 'bitmovin-player';
-import { i18n } from '../../localization/i18n';
+import { i18n, LocalizableText } from '../../localization/i18n';
+import { DynamicSettingsPanelItem } from '../dynamicsettingspanelitem';
+import { ListSelector, ListSelectorConfig } from '../listselector';
 
 /**
  * @category Configs
  */
-export interface SubtitleSettingsPanelPageConfig extends ContainerConfig {
+export interface SubtitleSettingsPanelPageConfig extends SettingsPanelPageConfig {
   settingsPanel: SettingsPanel;
   overlay: SubtitleOverlay;
+  useDynamicSettingsPanelItem?: boolean;
```

Comment:
We still don't know how we want to ship the new UI layout. We are currently leaning towards a new major version. Once this decision is confirmed I'll rework this and just change to the new dynamic approach and throw out the config option again. For now I didn't want to create a subclass for this as the code is very similar.